tests/layer_tests/pytorch_tests/test_pooling.py

```diff
@@ -19,8 +19,6 @@
              {'kernel_size': [2, 1], 'stride': [], 'padding': 0},
              ]
 
-d2_params_corner_case = [{'kernel_size': [8, 8], 'stride': [8, 4], 'padding': 1}]
```

Comment:
This case should be added to `d2_params` otherwise you do not check that it works